pkg/sip/participant.go

```diff
@@ -75,6 +75,7 @@ func (v CallStatus) DisconnectReason() livekit.DisconnectReason {
 
 const (
 	callDropped = CallStatus(iota)
+	callFlood
```

Comment:
This constant should not be exported?

Reply:
We don't distinguish it in participant attributes, so it's unexported.